functional-tests/Readme.md

```diff
@@ -14,28 +12,84 @@ BDDStack is 100% compatible with the tests that were created in the previous inc
 
 Please see the [Wiki](https://github.com/BCDevOps/BDDStack/wiki) for more details.
 
-## Usage
+[navunit]: https://github.com/bcgov/navUnit
+[dockerfile]: https://github.com/BCDevOps/openshift-tools/blob/master/provisioning/jenkins-slaves/bddstack/Dockerfile
+[issue_tracker]: https://github.com/rstens/BDDStack/issues
+[slack_channel]: https://devopspathfinder.slack.com/messages/C7J72K1MG
+
+# Usage
+
+The following commands will launch the tests with the individual browser:
+```
+./gradlew chromeTest
+./gradlew chromeHeadlessTest // Will run in OpenShift
+
+./gradlew firefoxTest
+./gradlew firefoxHeadlessTest // Will run in OpenShift
+```
+
+The following are experimental and may need additional work/configuration to make work:
+```
+./gradlew edgeTest // Only on Windows
+./gradlew ieTest // Only on Windows, read wiki for set up instructions
+./gradlew safariTest // Only on MacOS, read wiki for set up instructions
+```
+
+# Geb - Key Concepts
```

Comment:
This section is my attempt to make learning Geb easier.

It contains a list of key topics, and their sections in the documentation, that I think are most important to learn/reference initially.